src/pages/OperationsPage.js

```diff
@@ -47,10 +48,12 @@ class OperationsPage extends Component {
     }
 
     componentWillMount(){
-        this.props.getConfirmationsOnce();
-        this.props.getListCBE();
-        this.props.getPendingsOnce();
-        this.props.getLOCsOnce();
+        this.props.getConfirmationsOnce();   //  todo
+        this.props.getListCBE();   //  todo
+        this.props.getPendingsOnce();   //  todo
+        if (this.props.isLOCsNeedReload) {
```

Comment:
@Pavel-DV with this we can even extend base class of components with some kind of _prepare()_ function which will be called by super _componentDidMount_ when _isReady_ is false and which should contain calls of necessary action-creators and each of action may have it's own flag, but reducer will it to one when all such flags will be marked as true.

You continue to object or ask, so I keep on answering. And it's not my fad, I don't want you to do anything for my 'pleasure'...

I guess and hope we've decided to request from this PR:
1. Renaming all such flags to 'isReady' in all components.
2. Implementation of state clearing on SESSION_DESTROY.